doc/source/tools/pointwise/pointwise.rst

```diff
@@ -0,0 +1,335 @@
+
+.. image:: images/assemble_domains.png
+    :align: center
+
+.. warning::
+    A structured mesh is created by selecting a closed quadrilateral of **connectors**. The quadrilateral can contain more than 4 connectors. However, the opposing sides of that quadrilateral **must** have the same dimension. 
+
+- Repeat the process for the four sides of the first square. After the second border of the quadrilateral has been confirmed, the mesh should complete itself. Repeat for the other two squares. The final result should look like this.
+
+.. image:: images/assemble_domains2.png
+    :align: center
+
+.. warning::
+    If a mistake was made in the first steps, they generally manifest at this exact moment. Here are some common mistakes to look into if the mesh is not correctly generated.
+
+    - **Next Edge** was not selected before clicking on the next side of the square.
+    - The connectors are dimensionless.
+    - A line is not correctly connected to its neighboring line.
+    - The dimension of the connector on the opposite side do not match.
+    - A line was not split when the geometry was separated into subsections. 
+
+
+----
+
+Assigning the boundary conditions and exporting the mesh
+---------------------------------------------------------
+
+- In the **CAE** option, select **Set Dimension** and choose 2D.
+
+- Go to the **CAE** option and choose **Select Solver**. In the list on the left of the screen, select **GMSH**.
+
+- In the **CAE** option, select **Set Boundary Conditions**. Click **New**. Change the **Name** of the new box by double-clicking on it and insert **wall**. Select all the walls in the geometry (hold the ``ctrl`` button on your keyboard as you click) as is presented in the following image and click on the little check box at the left of the name box that was just changed. All the walls should be moved to that set of boundary conditions. The number 6 indicates that 6 boundaries were just moved to that set of boundary conditions. 
+
+.. image:: images/boundary_conditions.png
+    :align: center
+
+- Repeat the previous step for the inlet and outlet boundary conditions.
+
+.. image:: images/boundary_conditions2.png
+    :align: center
+
+- To export the 2D mesh, select all the domains. In the **file** option, select **export** and **CAE**. Choose a name for the .msh file and press save. Your mesh should be done!
+
+.. warning::
+    The Deal.II mesh file parser cannot read the comments in the ``.msh`` file generated by Pointwise. Make sure to delete manually all the commented information in the file before launching a simulation.
+ 
+The boundary conditions identification numbers are chosen by Pointwise. The user has no control over the software's decision. The boundary condition IDs can be retrieved at the end of the ``.msh`` file generated. In the present case, the boundary IDs would correspond to 2 for the inlet, 3 for the outlet and 4 for all the walls.
+
+.. image:: images/mesh_file.png
```

Comment:
Why is there one Unspecified?

Reply:
We did not specify te volume conditions. The solver will just assume it is the fluid